mkdocs.yml

```diff
@@ -111,6 +111,7 @@ nav:
           - Interfacing with LTS: data_management/lts/interfaces.md
           - Bucket Permissions: data_management/lts/policies.md
           - UAB Core Accounts: data_management/lts/lts_cores.md
+          - Top-level LTS Permissions: data_management/lts/top_level_lts_permission.md
```

Comment:
I'm wondering if we should merge this content into "Bucket Permissions" (policies.md) and rename that to "Access Control".

What are your thoughts?

Reply:
I agree. I have merged both contents and renamed "Bucket Permissions" to "Access Control".  Could you please review the merged content and let me know if you have any further feedbacks? Thanks.